vnext/ReactUWP/Utils/PropertyUtils.h

```diff
@@ -32,6 +32,16 @@ static double DefaultOrOverride(double defaultValue, double x) {
   return x != c_UndefinedEdge ? x : defaultValue;
 };
 
+static const std::unordered_map<std::string, ShadowEdges> edgeTypeMap = {
```

Comment:
>edgeTypeMap [](start = 58, length = 11)

If you are going to with map, I would suggest you put it in another PR. we have a lot of functions which can follow the same pattern (string -> enum) like padding

Reply:
I'd like to keep this change as for now, we can open a new issue to track creating consolidated property mapping utility as we have many property mappings code in various shadow node/view managers.

---
In reply to: [358372703](https://github.com/microsoft/react-native-windows/pull/3754#discussion_r358372703) [](ancestors = 358372703)